### PR TITLE
EOS-23002: Node Health - Resource health status is coming as 'NA' for 'inactive' services and other resources

### PR DESCRIPTION
**Pull request by sradhanand-pati**
Signed-off-by: sradhanandpati <sradhanand.pati@seagate.com>

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: lr-cli/cortx_setup/commands/resource/show.py [hunk 1]
```diff
@@ -81,8 +81,7 @@ def filter_resource_type(self, resource_type, resource_dict):
             'type': str,
             'default': None,
             'optional': True,
-            'choices': ['OK', 'FAULT', 'DEGRADED'],
-            'help': 'The current state of the resources'
+            'help': "The current state of the resources  like 'OK', 'FAULT', 'DEGRADED', 'NA', 'INACTIVE'"
```
Comment (nitin-seagate) [inline lr-cli/cortx_setup/commands/resource/show.py:84]: Add `UNKNOWN`

**Review by nitin-seagate — APPROVED** [review]
LGTM

File: lr-cli/cortx_setup/commands/resource/show.py (hunk 1 above)
Comment (sradhanand-pati) [reply lr-cli/cortx_setup/commands/resource/show.py]: Done

**Pull request merged**